.travis.yml

```diff
@@ -1,7 +1,230 @@
+      dist: trusty
+      language: python
+    - env: TESTENV=jython-dev DIST=trusty
+      dist: trusty
+      language: python
+
+    # Miniconda 2
+
+    - env: TESTENV=miniconda2-4.0.5 DIST=precise
+      python: 2.7.9
+      dist: precise
+    - env: TESTENV=miniconda2-4.0.5 DIST=trusty
+      python: 2.7.9
+      dist: trusty
+
+    # Stackless
+
+    - env: TESTENV=stackless-dev DIST=precise
+      dist: precise
+      language: python
+
+    # IronPython
+
+    - env: TESTENV=ironpython-2.7.5 DIST=precise
+      dist: precise
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+    - env: TESTENV=ironpython-dev DIST=precise
+      dist: precise
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+    - env: TESTENV=ironpython-dev DIST=trusty
+      dist: trusty
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+
+  allow_failures:
+    - env: TESTENV=pypy-dev DIST=precise
+      dist: precise
+      language: python
+    - env: TESTENV=pypy-dev DIST=trusty
+      dist: trusty
+      language: python
+
+    - env: TESTENV=jython-dev DIST=trusty
+      dist: trusty
+      language: generic
+
+    - env: TESTENV=jython-2.7.0 DIST=precise
+      dist: precise
+      language: python
+    - env: TESTENV=jython-dev DIST=precise
+      dist: precise
+      language: python
+    - env: TESTENV=jython-2.7.0 DIST=trusty
+      dist: trusty
+      language: python
+    - env: TESTENV=jython-dev DIST=trusty
+      dist: trusty
+      language: python
+
+    - env: TESTENV=miniconda2-4.0.5 DIST=precise
+      python: 2.7.9
+      dist: precise
+    - env: TESTENV=miniconda2-4.0.5 DIST=trusty
+      python: 2.7.9
+      dist: trusty
+
+    - env: TESTENV=stackless-dev DIST=precise
+      dist: precise
+
+    - env: TESTENV=ironpython-2.7.5 DIST=precise
+      dist: precise
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+    - env: TESTENV=ironpython-dev DIST=precise
+      dist: precise
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+    - env: TESTENV=ironpython-dev DIST=trusty
+      dist: trusty
+      sudo: required
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+            - mono-xbuild
+            - mono-dmcs
+            - mono-complete
+
+install:
+  - git clone --depth 1 https://github.com/sstephenson/bats.git
+  - |
```

Comment:
Ironpython install a binary `python` which needs to be run by mono, or use this trick on a sudo enable worker, which is slower than the precise containers. Otherwise virtualenv tries to run it, and fails.
Is it possible to detect this and install a script `python` which invokes mono?